source/features/preserve-whitespace-option-in-nav.tsx

```diff
@@ -15,7 +15,7 @@ function init() {
 		for (const a of navLinks) {
 			const linkUrl = new URL(a.href);
 			linkUrl.searchParams.set('w', '1');
-			a.href = linkUrl;
+			a.href = String(linkUrl);
```

Comment:
Not strictly related to types, but this can follow the style we used in another recent PR:

```js
const params = new URLSearchParams(a.search);
params.set('w', '1');
a.search = params.toString();
```

Reply:
Will do. I think you gave the option of one or the other, but happy to keep it uniform in your packages.

Reply:
will make the change.

Reply:
done

Reply:
Reverted it.